app/src/main/java/com/github/se/wanderpals/ui/screens/dashboard/DashboardSuggestionWidget.kt

```diff
@@ -63,131 +60,169 @@ fun DashboardSuggestionWidget(viewModel: DashboardViewModel, onClick: () -> Unit
+                        }
 
-          Spacer(modifier = Modifier.padding(6.dp))
+                    Spacer(modifier = Modifier.padding(4.dp))
 
-          if (sortedSuggestion.isEmpty() ||
-              sortedSuggestion.none {
-                it.stop.stopStatus == CalendarUiState.StopStatus.NONE
-              }) { // if there are no suggestions or if all suggestions are added to stops
-            Column(
-                modifier =
-                    Modifier.fillMaxWidth()
-                        .clip(RoundedCornerShape(10.dp))
-                        .background(backgroundLight)
-                        .padding(40.dp),
-                verticalArrangement = Arrangement.Center,
-                horizontalAlignment = Alignment.CenterHorizontally) {
-                  Text(
-                      text = "No suggestions available",
-                      style = TextStyle(fontSize = 16.sp, fontWeight = FontWeight.Bold),
-                      modifier = Modifier.testTag("noSuggestions"))
-                }
-          } else {
-            sortedSuggestion
-                .filter { it.stop.stopStatus == CalendarUiState.StopStatus.NONE }
-                .take(DISPLAY_COUNT)
-                .forEachIndexed { index, suggestion ->
-                  SuggestionItem(suggestion = suggestion)
-                  if (index < DISPLAY_COUNT - 1) {
-                    Spacer(modifier = Modifier.height(8.dp)) // Add space between items
+                    // List of suggestions
+                    Box(
+                        modifier =
+                            Modifier.clip(RoundedCornerShape(4.dp))
+                                .background(MaterialTheme.colorScheme.surface)
+                                .fillMaxWidth()) {
+                          if (sortedSuggestion.isEmpty()) {
+                            Box(
+                                contentAlignment = Alignment.Center,
+                                modifier =
+                                    Modifier.padding(top = 16.dp, bottom = 40.dp).fillMaxSize()) {
+                                  Text(
+                                      text = "No suggestions yet.",
+                                      modifier = Modifier.testTag("noSuggestions"),
+                                      style = TextStyle(color = MaterialTheme.colorScheme.primary),
+                                  )
+                                }
+                          } else {
+                            Column {
+                              SuggestionWidgetItem(suggestion = sortedSuggestion[0])
+                              HorizontalDivider(
+                                  color = MaterialTheme.colorScheme.surfaceVariant,
+                                  thickness = 1.dp,
+                                  modifier = Modifier.padding(horizontal = 8.dp))
+                              if (sortedSuggestion.size > 1) {
```

Comment:
You can use when conidition instead of nested if condition

Reply:
i can remove a nested if and it would make more sense but switching to a when would not do the same thing and would create a lot of duplication